core/trino-main/src/test/java/io/trino/server/security/oauth2/TestOAuth2WebUiAuthenticationFilter.java

```diff
@@ -181,10 +180,8 @@ public void testInvalidToken()
                 .signWith(signatureAlgorithm, keyGenerator.generateKeyPair().getPrivate())
                 .compact();
 
-        try (Response response = httpClient.newCall(
-                uiCall()
-                        .header(AUTHORIZATION, "Bearer " + token)
-                        .build())
+        try (Response response = httpClientUsingCookie(new Cookie.Builder("Presto-OAuth2-Token", token).build())
```

Comment:
Why not to use `OAUTH2_COOKIE`? 

Notice that cookie was renamed.

Reply:
I think this is should be changed in this commit.